controllers/userController.js

```diff
@@ -0,0 +1,108 @@
+const userService = require("../services/dbUserServices");
+
+class UserController {
+  async getAllUsers(req, res) {
+    try {
+      const users = await userService.getAllUsers();
+      if (!users) {
+        res.status(200).json({
+          status: "error",
+          message: "user didn't created yet",
+        });
+      }
+      res.status(200).json({
+        users,
+        status: "success",
+      });
+    } catch (e) {
+      res.status(500).json({
+        message: "unexpected server error",
+      });
+    }
+  }
+
+  async addNewUser(req, res) {
+    try {
+      const isUserCreated = await userService.addNewUser(req.body);
+      if (!isUserCreated) {
+        res.status(200).json({
+          status: "error",
+          message: "user has not been created",
+        });
+      }
+      res.status(200).json({
+        status: "success",
+        message: "user created",
+      });
+    } catch (e) {
+      res.status(500).json({
+        message: "unexpected server error",
+      });
+    }
+  }
+
+  async getUser(req, res) {
+    try {
+      const user = await userService.getUser(req.params.id);
+      if (!user) {
+        res.status(200).json({
```

Comment:
Look, I'd like you to investigate a bit about HTTP errors and cases of their use - https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design (a short example), because I reckon that 204 suits this case better